R/list_comparison_linter.R

```diff
@@ -21,8 +21,6 @@
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
 list_comparison_linter <- function() {
-  # TODO(michaelchirico): extend to cases where using simplify=FALSE implies a
```

Comment:
ditto

Reply:
Here I don't think the TODO anchors to this expression in particular, so I removed it. The issue is #2472.